app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -188,13 +188,59 @@
                                     :id                => record.id,
                                     :prefix            => "amqp",
                                     :basic_info_needed => true}
+
```

Comment:
I've attempted to merge with OpenStack tab, but there were too much ng-if statements for my taste. I'd prefer to go with some duplication for code to remain readable/managable.